unlock-app/src/components/interface/user-account/KeyPurchaseConfirmation.tsx

```diff
@@ -11,63 +11,93 @@ import {
   SubmitButton,
   LockInfo,
   DisabledButton,
+  LoadingButton,
 } from './styles'
 import { Fees } from '../../../actions/keyPurchase'
 
-interface KeyPurchaseConfirmationProps {
+interface Props {
   address: string
   emailAddress: string
   lock?: Lock
-  cards: stripe.Card[]
```

Comment:
Any reason this goes from multiple cards to a single card?

Reply:
Previously this component just pulled the first card (if present) out of the array and rendered it. Now `mapStateToProps` does that. The actual output is the same (though some stories need to be updated, since the stories use the unconnected component) but it makes the component easier to read.